api/app/views/spree/api/credit_cards/show.v1.rabl

```diff
@@ -1,3 +1,7 @@
 object @credit_card
 cache [I18n.locale, root_object]
 attributes *creditcard_attributes
+
+child :address do
```

Comment:
@athal7 great catch. Greg and I talked this over for a bit we think we're OK.  We are still not updating addresses in-place anywhere -- you would always create a new address via /admin or /api.  So that means the credit_card.address_id will change and the cache will get invalidated.  Theoretically someone could update an address directly but that shouldn't be possible through /api or /admin and @gvaughn's' upcoming immutable-address work should resolve even that scenario.  Seem good?